### DISCUSSION TITLE
How to get instance/semantic segmentation feature

### DISCUSSION THREAD

**Discussion by h-peng17** [facebookresearch/detectron2#2681] [Q&A]
Hi~ 
We can get object detection feature by running the model step by step like https://detectron2.readthedocs.io/en/latest/tutorials/models.html. But how can I get instance/semantic segmentation feature from models? The predicted `instances` has `pred_masks`, how can I use it? Wish for your reply~

**Comment by deepaksinghcv** [comment]
In https://detectron2.readthedocs.io/en/latest/tutorials/models.html#model-output-format, it is mentioned that in inference mode the model will output a list[dict] for each image with appropriate keys to access `pred_masks`
@ppwwyyxx Kindly correct me if I'm wrong.
